packages/cli-e2e/entrypoints/common.js

```diff
@@ -62,21 +63,22 @@ const ensureDockerImageIsPresent = () => {
 };
 
 const createEnvFile = () => {
-  const environmentVariables = [
-    'PLATFORM_USER_NAME',
-    'PLATFORM_USER_PASSWORD',
-    'GITHUB_ACTION',
-  ];
+  const environmentVariables = ['PLATFORM_USER_NAME', 'PLATFORM_USER_PASSWORD'];
 
   if (existsSync('.env')) {
     return;
   }
 
   writeFileSync(
     '.env',
-    environmentVariables
-      .map((variable) => `${variable}=${process.env[variable]}`)
-      .join('\n')
+    [
+      ...environmentVariables.map(
+        (variable) => `${variable}=${process.env[variable]}`
+      ),
+      `TEST_RUN_ID=${
+        process.env.GITHUB_ACTION || randomBytes(16).toString('hex')
```

Comment:
githubaction wont exist on your machine.  and you don't want to collide with other peeps running test on their machine too.
So if github_action ain't available, let's generate a pseudo-uuid and use it instead.